daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/RuntimePropertyMixins.scala

```diff
@@ -467,7 +467,7 @@ trait ElementRuntimeValuedPropertiesMixin
   private lazy val occursCountExpr = LV('occursCount) {
     val qn = this.qNameForProperty("occursCount")
     val isEvaluatedAbove = true
-    ExpressionCompilers.JLong.compile(qn, NodeInfo.Long, occursCountRaw, isEvaluatedAbove)
+    ExpressionCompilers.JLong.compile(qn, NodeInfo.Long, occursCountRaw, decl, isEvaluatedAbove)
```

Comment:
Shouldn't the decl parameter be after isEvaluatedAbove?

Reply:
Yeah, that little inconsistency is because one of the overloads of compile() has an optional final arg. I should probably eliminate that, or get rid of all this overloading and give them names like compilePropertyExpression,.....

Reply:
Fixed this in the latest commit. Refactored this confusing class, renamed methods so they're distinct by purpose of the method.